CHANGELOG.rst

```diff
@@ -1,6 +1,21 @@
 Change log
 ==========
 
+0.19.12 (2022-01-26)
+--------------------
+
+New features
+~~~~~~~~~~~~
+
+* **out:** convert ANSI escape sequences to HTML elements formatting
```

Comment:
I think for end users it is really hard to understand what is the change and what are the benefits.
The text doesn't even mention that it is in html log.